theano/tensor/nnet/nnet.py

```diff
@@ -2230,31 +2229,23 @@ class of the input x while the second predicts the output of the input x in
 
     The outputs are grouped in the same order as they are initially defined.
 
-    .. versionadded:: 0.7.1
+    .. versionadded:: 0.7.1 
 
     Parameters
     ----------
-    x: tensor of shape (batch_size, number of features)
+    x: tensor of shape (batch_size, number_of_features)
```

Comment:
I disagree. `batch_size` is a parameter and refers to a specific variable, not `number_of_features`.